src/query/functions/temporal/aggregation.go

```diff
@@ -87,13 +92,112 @@ func NewAggOp(args []interface{}, optype string) (transform.Params, error) {
 		return newBaseOp(args, optype, a)
 	}
 
+	if optype == HoltWintersType {
```

Comment:
move holtwinters into its own file, this logic can live there. 

if optype == HoltWintersType {
 return newHolterWintersOp(args)
}

Reply:
Hmm and have a separate `HoltWintersOp`?